### ISSUE TITLE
Image Size

### ISSUE THREAD

**Issue by comunevi** [libasys/pinit#10]
Good morning After uploading the image as you can see it for the whole pixel as I loaded?
##

**Comment by libasys** [comment]
i see no image ;)!

**Comment by comunevi** [comment]
For example: I PIN an image 2048x1311 pixel, but when i want to view It is resized 400xXXX pixel, it' possible i can see also in the original size?

**Comment by libasys** [comment]
at the moment: no!

**Comment by comunevi** [comment]
thank you, think is possible in the future version?

**Comment by libasys** [comment]
I don't think so, cause i see not the need for this? tell me an example where this is needed?

**Comment by comunevi** [comment]
for a photographic historian, we will use it internally in our offices not to disclosures
